superbench/analyzer/result_summary.py

```diff
@@ -217,7 +217,7 @@ def output_summary_in_excel(self, raw_data_df, summary, output_path):
                 file_handler.merge_column_in_excel(worksheet, row, 1)
             else:
                 logger.error('ResultSummary: excel_data_output - summary is empty.')
-            writer.save()
+            writer.close()
```

Comment:
So, it will save the excel when closing?

Reply:
yes, it's synonym for save, they replace save to close in latest version to make it more file-like